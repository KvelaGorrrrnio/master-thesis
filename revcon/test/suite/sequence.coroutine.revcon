@coroutine

coroutine main(int a) {
  a += 1
  local int b 10
  a *= b
  b -= 10
  delocal int b 0
}

