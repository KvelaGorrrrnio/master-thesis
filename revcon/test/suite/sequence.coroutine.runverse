test/suite/sequence.coroutine.revcon:
[store]
  a: 0
  main: Ã˜

[stack]
[spawn main main end () | unresume main (a) | despawn main main begin ()] <|> []

