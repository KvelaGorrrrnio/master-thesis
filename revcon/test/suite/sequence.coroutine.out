test/suite/sequence.coroutine.revcon:
[store]
  a: 10
  main: Ã˜

[stack]
[spawn main main begin () | resume main (a) | despawn main main end ()] <|> []

